src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -2,21 +2,51 @@
 Copyright (c) 2018 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Johannes Hölzl
-
-Continuous linear functions -- functions between normed vector spaces which are bounded and linear.
 -/
 import analysis.normed_space.multilinear
 
+/-!
+# Continuous linear maps
+
+This file defines a class stating that a map between normed vector spaces is (bi)linear and
+continuous.
+Instead of asking for continuity, the definition takes the equivalent condition that `∥f x∥` is
+bounded by a multiple of `∥x∥`. Hence the "bounded" in the name refers to `∥f x∥/∥x∥` rather than
+`∥f x∥` itself.
+
+## Main declarations
+
+* `is_bounded_linear_map`: Class stating that a map `f : E → F` is linear and continuous, but
+  through the simpler to provide statement that `∥ f x ∥` is bounded by a multiple of `∥ x ∥`.
+  Effectively an unbundled version of `continuous_linear_map`, albeit the name disparity.
```

Comment:
There are actually two differences:  the bundling/unbundling, and the fact that `continuous_linear_map` is more general (it works for topological vector spaces where there is no notion of boundedness).  So I think it's not quite right to write off the difference in naming as a quirk -- it's more fundamental.

Reply:
So what exactly is the point of asking for boundedness rather than continuity? I understand what you're saying, but why aren't we in the situation where we have `is_continuous_linear_map` + lemmas that prove it is equivalent to being bounded when the space is normed?

Reply:
Sorry for the slow reply.  I think the ideal state of things wold be to have neither `is_bounded_linear_map` nor `is_continuous_linear_map`, but only `continuous_linear_map`; it's for historical reasons that we have the former.  Does that make sense?

In terms of how we explain this, maybe just give a simple description here, and leave the explanation to the paragraphs above and below which you already wrote.
> * `is_bounded_linear_map`: Class stating that a map `f : E → F` is linear and has `∥f x∥` bounded by a multiple of `∥x∥`.